src/gui/Src/Gui/MainWindow.h

```diff
@@ -182,7 +180,6 @@ public slots:
     DebugStatusLabel* mStatusLabel;
     LogStatusLabel* mLastLogLabel;
 
-    UpdateChecker* mUpdateChecker;

```

Comment:
I remember there was a reason why this `UpdateChecker` was stored on the heap. I think it's a good idea to replace this with an `AboutDialog*` to make sure no weird bugs are introduced.